XamarinCommunityToolkitSample/ViewModels/WelcomeViewModel.cs

```diff
@@ -1,16 +1,15 @@
-﻿using XamarinCommunityToolkitSample.Models;
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using XamarinCommunityToolkitSample.Models;
 
 namespace XamarinCommunityToolkitSample.ViewModels
 {
     public class WelcomeViewModel : BaseViewModel
     {
-        public WelcomeSectionModel[] Items { get; } =
-        {
-            new WelcomeSectionModel(WelcomeSectionId.Behaviors),
-            new WelcomeSectionModel(WelcomeSectionId.Converters),
-            new WelcomeSectionModel(WelcomeSectionId.Extensions),
-            new WelcomeSectionModel(WelcomeSectionId.Views),
-            new WelcomeSectionModel(WelcomeSectionId.TestCases)
-        };
+        public IEnumerable<WelcomeSectionModel> Items { get; } =
+            ((WelcomeSectionId[])Enum
+            .GetValues(typeof(WelcomeSectionId)))
+            .Select(id => new WelcomeSectionModel(id));
```

Comment:
I see this code mostly duplicated three times now. Can we fix that somehow? Also, it seems a bit complicated for what it does, but that might just be me

Reply:
Unfortunately, that's impossible.

1) Generic constraints don't allow to specify constructors with parameters. We won't be able to pass id there if we'd like to extract this logic into separate class.